### PR TITLE
Delve Debugger vs (and just) Delve

### PR DESCRIPTION
**Pull request by codeputer**
As a windows focused developer, Delve means something very different for me, and I thought a sentence to clarify the purpose of Delve for Go on the configuration page would immediately add context and reduce confusion

As I always strive to do in any documentation - NOT the what, but the WHY is more important.  If you understand WHY the feature was developed or used, the WHAT it does and HOW starts in the right context of purpose.

# Description

Add context to the "Delve" keyword, as I found it confusing as a windows focused developer

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation
* [ ] Specification has been updated
* [ ] Provided sample for the feature

### CODE REVIEW

File: docs/development/setup-dapr-development-env.md [hunk 1]
```diff
@@ -30,7 +30,8 @@ This document helps you get started developing Dapr. If you find any problem whi
    GOPATH=c:\go
    PATH=%GOPATH%\bin;...
    ```
-2. [Delve](https://github.com/go-delve/delve/tree/master/Documentation/installation) for Debugging
+2. [Delve](https://github.com/go-delve/delve/tree/master/Documentation/installation) for GO debugging
```
Comment (yaron2) [inline docs/development/setup-dapr-development-env.md]: nit: Maybe replace `GO` with `Go` for consistency..

**Comment by youngbupark** [comment]
@yaron2 please review this pr again.

**Comment by yaron2** [comment]
LGTM

**Pull request merged**